src/frontend/src/pages/HomePage/components/GuestOrganizationInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import { Card, Typography, useTheme } from '@mui/material';
+import { Grid } from '@mui/material';
+import { useCurrentOrganization } from '../../../hooks/organizations.hooks';
+import React from 'react';
+import { NERButton } from '../../../components/NERButton';
+import * as Icons from '@mui/icons-material';
+import { useAllLinkTypes, useAllUsefulLinks } from '../../../hooks/projects.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { Stack } from '@mui/system';
+
+interface GuestOrganizationInfoButtonProps {
+  href?: string;
+  buttonText: string;
+  iconName: string;
+}
+
+const NERGuestButton: React.FC<GuestOrganizationInfoButtonProps> = ({ href, buttonText, iconName }) => {
+  const IconComponent = Icons[iconName as keyof typeof Icons] || Icons['Info'];
+
+  return (
+    <Grid item xs={4}>
+      <NERButton variant="contained" fullWidth={true} style={{ justifyContent: 'flex-start', color: 'white' }} href={href}>
+        <IconComponent />
```

Comment:
Instead of this just surround the iconName with the mui/material icon component like this:
<Icon>{iconName}</Icon>